pipelines/_ingredients/ApPipeWithFakes.yaml

```diff
@@ -13,7 +13,7 @@ description: AP Pipeline with synthetic/fake sources. Templates are inputs.
 # pipeline, e.g., $AP_PIPE_DIR/pipelines/$CAMERA/ApPipeWithFakes.yaml.
 
 imports:
-  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcd.yaml
+  - location: $AP_PIPE_DIR/pipelines/_ingredients/ProcessCcdCalibrate.yaml
```

Comment:
Add a comment saying that the old fakes system isn't really compatible with `CalibrateImage`?